lib/client.js

```diff
@@ -76,4 +77,39 @@ Client.prototype.listArchives = function() {
 Client.prototype.deleteArchive = function() {
 };
 
+Client.prototype.dial = function (opts, cb) {
+  request.post({
+    // TODO: only works while apiUrl is always up to the domain, without the ending slash
+    url: this.c.apiUrl + '/v2/project/' + this.c.apiKey + this.c.endpoints.dial,
+    json: opts,
+    headers: {
```

Comment:
we should probably refactor this into something that both `createSession()` and `dial()` can use in common.